src/schema/root.js

```diff
@@ -94,7 +94,7 @@ type Query {
   """
   localizationsWithHoldings parses ALL localizations and ALL detailedholdings. Returns agencies with holdings on shelf
   """
-  localizationsWithHoldings(pids: [String!]!, limit: Int, offset: Int): Localizations 
+  localizationsWithHoldings(pids: [String!]!, lookupSpecificAgencyIds: [String!], shuffleLocalizations: Boolean limit: Int, offset: Int): Localizations 
```

Comment:
hvad gøre shuffle dimsen .. og hvorfor giver du lookupspecificagencyid i argumenter .. er det ikke noget som skal håndteres under alle omstændigheder ?

Reply:
- Shuffle-dimsen blander rækkefølgen på agencyIds, for ellers er det altid øen, kbh, etc. som vises først, og jeg tænkte at det ville være bedre bare at vise en tilfældig rækkefølge, så bibliotekerne ikke bliver sure. (Kan jeg godt fjerne hvis det er, men jeg tænker det er rimeligt, at bibliotekerne bliver repræsenteret ligeligt på Bibliotek.dk)
- lookupspecificagencyid: altså det var det jeg prøvede at sige efter daily: At jeg har brug for, at kunne tjekke specifikke localizationsWithHoldings på specifikke agencyId, og om det var okay at jeg tilføjede sådan et filter. Hvad mener du med "er det ikke noget som skal håndteres under alle omstændigheder"?